simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { ShareButton } from '../../common/Button/Button.stories';
+
+const CopyQueue = (props) => {
+  const [clicked, setClicked] = React.useState(false);
+  const shareUrl = `${window.location.origin}/j/${props.queueId}`;
+
+  const handleShareButtonClick = () => {
+    setClicked(true);
+    setTimeout(() => setClicked(false), 5000); // changes back to old text and color after 5 seconds
```

Comment:
let's change back after 5 minutes?

let's do this:
```
const SHARE_BUTTON_RESET_TIMEOUT = 5 * 60 * 1000; // 5 minutes
...
  setTimeout(() => setClicked(false), SHARE_BUTTON_RESET_TIMEOUT);
```

Reply:
Why that long? My idea was this is like a tooltip, just to indicate that copy worked.

Reply:
Hmm. Leave it for now. We can revisit this later. 

I was thinking that once you copy, it will stay like that for ever. Then 5 minutes later, let it silently come back. We could check with @akashkbaburajan later.